server/admin/tree_gc.go

```diff
@@ -40,6 +40,9 @@ var (
 
 	hardDeleteCounter monitoring.Counter
 	metricsOnce       sync.Once
+
+	optsRead  = storage.NewGetOpts(storage.Admin, true)
```

Comment:
`Admin` seems the only type that can have `Readonly` true of false?

Reply:
Probably true at the moment but I'm not 100% convinced yet. Let's revisit when mirroring is done. Also there is some value in being able to catch errors where someone said they wanted a read access but then tried to write. Not sure how much value atm though.